docs/proposals/custom-wbt-settings.md

```diff
@@ -0,0 +1,50 @@
+# Proposal: Support option to change WBT settings
+
+Last updated: 07.12.2021
+
+
+## Abstract
+
+CSI should have an opportunity to automate WBT (writeback throttling) disabling for specific Volumes.
+
+## Background
+
+Some Storage Systems, which use Baremetal CSI, may have performance issues due to incorrect WBT settings.
+So it is specific for each device (sdb, sdc, ...), CSI should have an opportunity to control it on Volume or Pod creation stage.
+
+## Proposal
+
+#### Operator part
+
+Add parameters to csi-baremetal-deployment helm chart
+
+- `WBTDisableOnHDDSC` -> set `WBTDisable=on` for `csi-baremetal-sc-hdd`
+- `WBTDisableOnSSDSC` -> set `WBTDisable=on` for `csi-baremetal-sc-ssd`
+- `WBTDisableOnNVMESC` -> set `WBTDisable=on` for `csi-baremetal-sc-nvme`
+
+#### Node Part
+
+- Node Stage ->
+ 1. Check SC Parameter for `WBTDisable=on`
```

Comment:
I suggest to have `wbt_lat_usec` and value set to 0 for disabling. User might want to play with the parameters.

Reply:
fixed